index.html

```diff
@@ -279,6 +282,42 @@ <h2 class="contact">Abdulrahmansh75@gmail.com</h2>
         src="images/illustration-contact-form.png"
         alt="contact"
       />
-    </div>
+      <img class="vector-desk" src="images/about-me-desk.png" alt="vector" />
+      <img class="vector-desk2" src="images/about-me-desk2.png" alt="vector" />
+      <img class="vector-desk3" src="images/desk-left.png" alt="vector" />
+      <img class="vector-desk4" src="images/desk-mid.png" alt="vector" />
+      <img class="vector-desk5" src="images/desk-right.png" alt="vector" />
+      <div class="line"></div>
+      <div></div>
+      <footer class="footer">
+        <ul class="footer-social">
+          <li>
+            <a href="#" class="social-link"
+              ><img src="images/twitter.png" alt="twitter"
+            /></a>
+          </li>
+          <li>
+            <a href="#" class="social-link"
+              ><img src="images/Linkedinicon.png" alt="Linkedinicon"
+            /></a>
+          </li>
+          <li>
+            <a href="#" class="social-link"
+              ><img src="images/medium.png" alt="medium"
+            /></a>
+          </li>
```

Comment:
Hello @abdulrahmanshr75  and @aeon9300, the job you have done so far on this project is clearly visible 💪🏻. Kudos to you 🍾. I however will require some small changes to make this portfolio perfect 💯 .

- [ ] I have noticed that the social icons at the bottom have almost no space between them and I think it will really be nice if you added some bit of padding to allow some breathing space. It will also help the users who would want to visit your social handles to click on them without getting worked out 🥺.

👉🏻 You can also compare designs below to see how much padding to add.

| Your design | Figma design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/4247740/145658189-bb29dd4f-5e9d-4868-8bd7-9455cca1b993.png) | ![image](https://user-images.githubusercontent.com/4247740/145658210-b1fffc5a-b95a-4a47-8959-e35943545517.png) |